response_operations_ui/views/logout.py

```diff
@@ -14,5 +14,10 @@ def logout():
     except KeyError:
         pass
 
+    try:
+        del (session['next'])
```

Comment:
Could we add this into the other try?

Reply:
I originally thought that but was thinking that it could just ignore the del(next) if for some reason survey wasn't there. it would just cause an exception and pass through

Reply:
Okay i'll add that change now